helper/funcs.go

```diff
@@ -12,6 +15,12 @@ import (
 	"github.com/hashicorp/hcl/hcl/ast"
 )
 
+const (
```

Comment:
Would it make sense to have the copied functions be in a separate file with a note of origin and sha.  Would help us track future changes/bug fixes.

Reply:
Agreed, this is done.